packages/design-tokens/scripts/formats.js

```diff
@@ -0,0 +1,38 @@
+const SD_lessVariables = require("../../../node_modules/style-dictionary/lib/common/formats")[
+  "less/variables"
+];
+
+const dateHeader = () => `
+  /**
+   * Do not edit directly
+   * Generated on ${new Date()}
+   */
+`;
+
+const lessVariables = {
+  name: "less/variablesCustom",
```

Comment:
Is there a specific name we could give it? What is custom about our way of formatting it?

Reply:
I meant to remove this. I was doing a trying a slightly different format than style-dictionary gives you by default